src/commands/general/cmd.help.ts

```diff
@@ -1,6 +1,18 @@
+// License: GPL-3
+// Maintainer: Shadofer#0001
+// Contributors: Otterlord#3653, KittyBorgX#1415
+//
+// File description:
+//     The file for the !help command of the TowerBot
+// 
+// Extra info:
+//     Displays all the 
+//     Makes request to wss://towerverse-beta.herokuapp.com'
+//
```

Comment:
0. Do we really need these in the files?
1. The maintainer is not shadofer
2. Needs to use github usernames